### PR TITLE
Added a few words needed

### PR DESCRIPTION
**Pull request by micaelis72**

### CODE REVIEW

File: styles/Vocab/Stakater/accept.txt [hunk 1]
```diff
@@ -37,6 +39,7 @@ HAProxy
 [Rr]epo[s]?
 [Rr]eusability
 [Rr]ole[Bb]inding
+[Ss]afespring
```
Comment (karl-johan-grahn) [inline styles/Vocab/Stakater/accept.txt]: only use `Safespring`

File: styles/Vocab/Stakater/accept.txt [hunk 2]
```diff
@@ -4,6 +4,8 @@
 [Aa]utoscale
 [Aa]utoscalers
 [Aa]utoscaling
+[Bb]aremetal
+bare-metal
```
Comment (karl-johan-grahn) [inline styles/Vocab/Stakater/accept.txt]: why not just standardize on baremetal?

File: styles/Vocab/Stakater/accept.txt [hunk 3]
```diff
@@ -99,6 +102,7 @@ EOC
 eppn
 ESXi
 etcd
+ETCD
```
Comment (karl-johan-grahn) [inline styles/Vocab/Stakater/accept.txt]: etcd is only lowercase

**Review by karl-johan-grahn — CHANGES_REQUESTED** [review]

File: styles/Vocab/Stakater/accept.txt (hunk 2 above)
Comment (micaelis72) [reply styles/Vocab/Stakater/accept.txt]: I've seen  both in lots of places, but it can leave just one, as standardization for Stakater's documents

**Review by karl-johan-grahn — APPROVED** [review]

**Pull request merged**